Assets/Scripts/JobSystem/JobsUI.cs

```diff
@@ -6,21 +6,20 @@
 
 public class JobsUI : MonoBehaviour
 {
+    public GameObject mainCanvas; 
+    public GameObject scheduleSlotPrefab; 
     public JobsContainer jobsContainer;
-    public GameObject jobPrefab; 
+    public Schedule schedule; 
+
     private GameObject availableJobsContainer;
     private GameObject scheduleContainer;
 
     private Sprite containerBackgroundSprite;
     private Sprite scheduleSlotSprite; 

```

Comment:
unused

Reply:
It is used to initialise the schedule slots: `for (int i = 1; i <= schedule.numberOfDays; i++)` 

But nowhere else it seems. I could move numberOfDays to a static member of a constants class or have it as a member of JobsUI. At the moment, it is stored in the schedule scriptableobject. 

What should I do with it?

Reply:
oh, i did a `find all references` and it didn't find any usages, must be a VS bug